packages/opencensus-core/src/metrics/metric-registry.ts

```diff
@@ -198,7 +198,13 @@ class MetricProducerForRegistry extends BaseMetricProducer {
    * @returns {Metric[]} The list of metrics.
    */
   getMetrics(): Metric[] {
-    return Array.from(
-        this.registeredMetrics, ([_, meter]) => meter.getMetric());
+    const metricArray = [];
```

Comment:
What would you think about rewriting this loop as `return this.registeredMetrics.values().map(m => m.getMetric()).filter(m => !!m);`?

Reply:
Done